wordpress-mobile/WordPressUI-iOS
# Adds optional bottom switch to FancyAlertView

### Description:

This PR adds an optional bottom switch to `FancyAlertView`.  This is intended to be used (at least initially) as a "Do not show this again" option for the alert.

Also adds 2 demos to showcase the "More information" button and the newly added switch.

Also fixes all unit tests.

### Testing:

Make sure the code looks fine.
Make sure the unit tests run fine.
Run the example App and:
- Open all 3 Fancy alert demos, and make sure they look fine.
- Try interacting with things and make sure an alert comes up describing your action.
